style.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.cdnfonts.com/css/google-sans');
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Product Sans', sans-serif;
+}
+body{
+    background-color: #5e0ff6;
+    background-image: url('../img/bg.png');
+}
+#header{
+    margin-top: 40px;
+}
+#header h2{
+    color: #fff;
+    text-align: center;
+}
+#header p{
+    text-align: center;
+    color: #fff;
+}
+#content label{
+    color: #fff;
+}
+#content input{
+    background-color: #6e26f4;
+    border: none;
+    padding: 15px;
+    color: #fff;
+    box-shadow: none;
+    margin-bottom: 15px;
+}
+#content textarea{
+    background-color: #6e26f4;
+    border: none;
+    padding: 15px;
+    color: #fff;
```

Comment:
combine multiple selectors with the same color #fff into one compound selector, you can separate the selectors with a comma and apply the color declaration after the compound selector, Using compound selectors can help to simplify your CSS code and make it easier to manage, especially when you have multiple selectors that share the same properties. It can also improve performance, as the browser only needs to apply the styles once for all the selectors in the compound selector.

Reply:
> - [ ] You did great work here👍Please kindly ensure to phrase the comments politely which should be clear and easy to understand. This really help to maintain a positive and productive atmosphere, which is crucial for collaboration.😊 
>
> > **You can simply say:** _I think it would be better to separate the selectors with a comma and apply the color declaration after the compound selector.....give explanation why the change._